src/backend/distributed/executor/adaptive_executor.c

```diff
@@ -1049,17 +1069,8 @@ ExecuteTaskListIntoTupleDest(RowModifyLevel modLevel, List *taskList,
 							 TupleDestination *tupleDest,
 							 bool expectResults)
 {
-	int targetPoolSize = MaxAdaptiveExecutorPoolSize;
-	bool localExecutionSupported = true;
-	ExecutionParams *executionParams = CreateBasicExecutionParams(
-		modLevel, taskList, targetPoolSize, localExecutionSupported
-		);
-
-	executionParams->xactProperties = DecideTransactionPropertiesForTaskList(
-		modLevel, taskList, false);
-	executionParams->expectResults = expectResults;
-	executionParams->tupleDestination = tupleDest;
-
+	ExecutionParams *executionParams = GetExecutionParams(modLevel, taskList, tupleDest,
+														  expectResults, NULL);
```

Comment:
We usually assign default values to variables and then pass them down to the functions:

```suggestion
	ParamListInfo paramListInfo = NULL;
	ExecutionParams *executionParams = GetExecutionParams(modLevel, taskList, tupleDest,
														  expectResults, paramListInfo);
```